internal/comment/handler.go

```diff
@@ -117,7 +117,7 @@ func (h *CommentHandler) CommentWithBehavior(ctx context.Context, behavior, body
 			break
 		}
 
-		logging.Logger.WithError(err).Debugf("received an error trying to post comment pausing %d seconds then will retry", backoff.Seconds())
+		logging.Logger.WithError(err).Debugf("received an error trying to post comment pausing %v seconds then will retry", backoff.Seconds())
```

Comment:
```suggestion
		logging.Logger.WithError(err).Debugf("received an error trying to post comment pausing %2f seconds then will retry", backoff.Seconds())
```

Reply:
Any reason why? I think I prefer `%v` or `%g` for these cases so a `float64(1.0)` just gets printed as `1` instead of `1.00`.

Reply:
no worries - %v just uses reflection so it's slightly slower, but sounds good

Reply:
![image](https://github.com/infracost/infracost/assets/158824/69cfd2b9-c071-4cd7-bec0-54de1eca22b3)